pkg/graveler/graveler.go

```diff
@@ -1904,7 +1905,7 @@ func (g *Graveler) isSealedEmpty(ctx context.Context, repository *RepositoryReco
 	if len(branch.SealedTokens) == 0 {
 		return true, nil
 	}
-	itrs, err := g.sealedTokensIterator(ctx, branch)
+	itrs, err := g.sealedTokensIterator(ctx, branch, 0)
```

Comment:
```suggestion
	itrs, err := g.sealedTokensIterator(ctx, branch, 1)
```

Reply:
Thanks - good call to optimize these calls too.